js/iframe-coordinator/elements/x-ifc-frame.ts

```diff
@@ -12,34 +12,41 @@ class ClientFrame extends HTMLElement {
 
   constructor() {
     super();
+  }
+
+  connectedCallback() {
```

Comment:
Wow!  I could see appending, but not even standard DOM apis?  Interesting and a good lesson learned.